### DISCUSSION TITLE
1 million downloads on Crates.io!

### DISCUSSION THREAD

**Discussion by sagebind** [sagebind/isahc#359] [Announcements]
Isahc has reached over 1 million downloads this week on Crates.io!

![Stats](https://user-images.githubusercontent.com/2192863/138615927-4da46a3a-7009-4f96-a57f-e68ccfd72874.png)

Thanks to everyone who's contributed to Isahc's success, whether code or examples or finding issues. I never thought that Isahc would become this popular and I am pleased to hear that people have found the library to be useful to them.

Hopefully Isahc continues to be useful to you into the future, as we have some long-awaited features in the works, so stay tuned!

**Comment by suryavirkapur** [comment]
Congratulations 🥳

isahc is my go to http client.

**Comment by mnj** [comment]
I recently started using this client, it's so nice to have a library that builds on libcurl, and it's many fixes to various quirks over the years.

Kept running into issues with the other famous http client in Rust, where you constantly had to disable built-in features and do them myself, because it didn't play well with some bad server implementations out there, even for plain standard features as following redirects. (was utterly broken in the other client).

So thanks dev and contributors :)
